schema/artwork/index.js

```diff
@@ -87,11 +87,10 @@ const ArtworkType = new GraphQLObjectType({
             description: 'Use whatever is in the original response instead of making a request',
           },
         },
-        resolve: ({ artists }, { shallow }) => {
+        resolve: ({ artists }, { shallow }, request, { rootValue: { artistsLoader } }) => {
           if (shallow) return artists;
-          return Promise.all(
-            artists.map(artist => gravity(`/artist/${artist.id}`))
-          ).catch(() => []);
+          const ids = artists.map(artist => artist._id);
+          return artistsLoader(null, { ids }).catch(() => []);
```

Comment:
We should defer this until we add batching, but ideally this would be something like the following where we _always_ pass in the ID as the first arg in a standardised way and let the decision of how to then actually fetch that data up to the implementation of the loader.

```js
return Promise.all(
  artists.map(({ id }) => artistLoader(id));
).catch(() => []);
```